gapic/src/main/com/google/gapid/perfetto/views/MemorySummaryPanel.java

```diff
@@ -121,9 +122,9 @@ protected void renderTrack(RenderContext ctx, Repainter repainter, double w, dou
         double dy = hovered.allSize.h / 4;
         ctx.setBackgroundColor(colors().background);
         ctx.fillRect(x, y + 1 * dy + (dy - LEGEND_SIZE) / 2, LEGEND_SIZE, LEGEND_SIZE);
-        ctx.setBackgroundColor(BaseColor.LIGHT_BLUE.rgb);
+        memoryBuffersGradient().applyBase(ctx);
         ctx.fillRect(x, y + 2 * dy + (dy - LEGEND_SIZE) / 2, LEGEND_SIZE, LEGEND_SIZE);
-        ctx.setBackgroundColor(BaseColor.PACIFIC_BLUE.rgb);
+        memoryUsedGradient().applyBase(ctx);
```

Comment:
I find that the first time I see this line, it's hard for me to notice there's a color change to ctx. Although I adjust to this new framework quickly and get to understand it, it may not be very friendly to a person new into the UI world? Maybe the name can give some hint about color?

Reply:
I was hoping having the workd "Gradient" in the function name would be enough...?